models/moco/hubconf.py

```diff
@@ -127,9 +121,6 @@ def train(self, niterations=1):
 
         Leave warmup to the caller (e.g. don't do it inside)        
         """
-        if self.device != "cuda":
```

Comment:
i'm confused - above you raise on CPU, but here you allow it?

Reply:
sorry I'm trying to do this quickly in between some other work, this is a typo.

Reply:
hah you and me both. i know the feeling..